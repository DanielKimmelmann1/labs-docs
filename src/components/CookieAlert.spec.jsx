import React from 'react';
import renderer from 'react-test-renderer';
import CookieAlert from './CookieAlert';

// LeftNav uses useStaticQuery which requires the data to be mocked in /__mocks__/gatsby.js

describe('Cookie Notification', () => {
  it('displays on initial load', () => {
    const tree = renderer
      .create(<CookieAlert />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
