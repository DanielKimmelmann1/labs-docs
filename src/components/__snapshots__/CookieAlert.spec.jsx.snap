// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cookie Notification displays on initial load 1`] = `
<div
  id="CookieDiv"
>
  <div
    className="CookieWrapper"
  >
    <div
      className="CookieConsent"
      style={
        Object {
          "alignItems": "baseline",
          "background": "white",
          "bottom": "0",
          "boxShadow": "0px 4px 26.91px rgba(0, 0, 0, 0.15)",
          "color": "#282828",
          "display": "block",
          "flexWrap": "wrap",
          "fontSize": "16px",
          "justifyContent": "space-between",
          "left": "0",
          "lineHeight": "24px",
          "padding": "20px",
          "position": "static",
          "textAlign": "center",
          "width": "100%",
          "zIndex": "999",
        }
      }
    >
      <div
        className=""
        style={
          Object {
            "flex": "1 0 300px",
            "margin": "15px",
          }
        }
      >
        We use cookies to analyze traffic, track and improve your experience, and assist in our marketing efforts. Our Cookie Notice provides more information and explains how to amend your cookie settings.
         
        <strong>
          <a
            href="https://www.postman.com/legal/cookies/"
            style={
              Object {
                "color": "black",
                "textDecoration": "underline",
              }
            }
          >
            Learn more
          </a>
        </strong>
      </div>
      <div
        className=""
      >
        <button
          className=""
          id="rcc-confirm-button"
          onClick={[Function]}
          style={
            Object {
              "background": "#1C272B",
              "border": "0",
              "borderRadius": "3px",
              "boxShadow": "0px 4px 4px rgba(0, 0, 0, 0.15)",
              "color": "#fff",
              "cursor": "pointer",
              "display": "block",
              "flex": "0 0 auto",
              "fontWeight": "500",
              "margin": "0 auto",
              "padding": "13px 24px",
              "textAlign": "center",
            }
          }
        >
          Accept Cookies
        </button>
      </div>
    </div>
  </div>
</div>
`;
